crates/bevy_app/src/app.rs

```diff
@@ -62,14 +63,38 @@ pub struct App {
     /// A container of [`Stage`]s set to be run in a linear order.
     pub schedule: Schedule,
     sub_apps: HashMap<AppLabelId, SubApp>,
+    /// Whether or not the App has been run and/or updated yet.
+    #[cfg(debug_assertions)]
+    pub has_run: bool,
 }
 
 /// Each `SubApp` has its own [`Schedule`] and [`World`], enabling a separation of concerns.
 struct SubApp {
     app: App,
     runner: Box<dyn Fn(&mut World, &mut App)>,
 }
-
+#[cfg(debug_assertions)]
```

Comment:
```suggestion

#[cfg(debug_assertions)]
```

Adding a blank line between `struct SubApp` and the attribute for `impl Drop for App`.